snuba/migrate.py

```diff
@@ -14,11 +34,84 @@ def _run_schema(conn, schema):
         return
     clickhouse_table = schema.get_local_table_name()
 
+    STRIP_CAST_RE = re.compile(r"^CAST\((.*), (.*)\)$", re.IGNORECASE)
+    FIXED_STRING_RE = re.compile(r"^FixedString\((\d+)\)$")
+    ENUM_RE = re.compile(r"^Enum\d+\((.*)\)$")
+    ENUM_VALUES_RE = re.compile(r"'(.+)' = (\d+)$")
+    MODIFIER_COL_RE = re.compile(r"^(Array|LowCardinality|Nullable)\((.+)\)$")
+
+    def strip_cast(default_expr: str) -> str:
+        match = STRIP_CAST_RE.match(default_expr)
+        if match:
+            default_expr = match.groups()[0]
+        return default_expr
```

Comment:
default_expr is not declared if the regex did not match anything. Did mypy complain ?

Reply:
It just returns the untransformed default_expr if the regex doesn't match